frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -29,8 +29,8 @@ export const colors = {
   "admin-navbar": "#7172AD",
   white: "#FFFFFF",
   success: "#84BB4C",
-  danger: "#ED6E6E",
-  error: "#ED6E6E",
+  danger: "hsla(358, 71%, 62%, 1)",
```

Comment:
we need raw values here as it's called in places that expect color value, not CSS custom properties.

Reply:
Can you at least leave a comment with the semantic names in JS?